core/modes/sleep_mode_pop_twice_to_wake.py

```diff
@@ -0,0 +1,22 @@
+import time
+
+from talon import Context, actions, settings
+
+ctx = Context()
+
+ctx.matches = r"""
+mode: sleep
+"""
+
+time_last_pop = 0
+
+
+@ctx.action_class("user")
+class UserActions:
+    def noise_trigger_pop():
+        global time_last_pop
+        if settings.get("user.mouse_enable_pop_wake") == 1:
+            delta = time.time() - time_last_pop
```

Comment:
wouldnt this raise an error the first time around since `time_last_pop` hasnt been defined with a value yet? there is only its global declaration without any associated value. Not sure if I am missing something...

Reply:
nevermind, I am stupid and hadnt seen line 11